app.js

```diff
@@ -41,18 +52,40 @@ app.use((req, res, next) => {
   return next();
 });
 app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({
-  limit: '30mb',
-  extended: true,
-}));
+app.use(
+  bodyParser.urlencoded({
+    limit: "30mb",
+    extended: true
+  })
+);
 
 app.use(routes);
 
+app.use("/auth", authRoutes);
+
+app.use("/profile", profileRoutes);
+
+app.use("/admin", adminRoutes);
+
 //routes
-app.get("/",(req,res)=>{
-   res.render("login.ejs");
+app.get("/", (req, res) => {
+  res.render("login.ejs");
 });
 
+app.post("/request/new", (req, res) => {
```

Comment:
Have this in some router inside routes. Prolly u can rename profile router to user.js and have these routes inside it.